### PR TITLE
pythonPackages.check-manifest,pythonPackages.swaglyrics: Fix dependencies; botan: fix compilation on darwin

### PR DESCRIPTION
**Pull request by SuperSandro2000**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/development/python-modules/ufoprocessor/default.nix [hunk 1]
```diff
@@ -26,6 +26,10 @@ buildPythonPackage rec {
     mutatormath
   ];
 
+  # no tests
```
Comment (sternenseemann) [inline pkgs/development/python-modules/ufoprocessor/default.nix:29]: Tests can be run using `python Tests/tests.py` according to source.

**Comment by siraben** [comment]
```
$ NIXPKGS_ALLOW_UNSUPPORTED_SYSTEM=1 nix-build -A swaglyrics
...
adding 'SwSpotify/__main__.py'
adding 'SwSpotify/spotify.py'
adding 'SwSpotify/web_server.py'
adding 'SwSpotify-1.2.1.dist-info/LICENSE'
adding 'SwSpotify-1.2.1.dist-info/METADATA'
adding 'SwSpotify-1.2.1.dist-info/WHEEL'
adding 'SwSpotify-1.2.1.dist-info/top_level.txt'
adding 'SwSpotify-1.2.1.dist-info/RECORD'
removing build/bdist.macosx-10.6-x86_64/wheel
Finished executing setuptoolsBuildPhase
installing
Executing pipInstallPhase
/private/tmp/nix-build-python3.8-SwSpotify-1.2.1.drv-0/source/dist /private/tmp/nix-build-python3.8-SwSpotify-1.2.1.drv-0/source
Processing ./SwSpotify-1.2.1-py3-none-any.whl
ERROR: Could not find a version that satisfies the requirement pyobjc; platform_system == "Darwin" (from swspotify)
ERROR: No matching distribution found for pyobjc; platform_system == "Darwin"
builder for '/nix/store/xm1li0fk77mbs20p1ka1h9145fixkxbk-python3.8-SwSpotify-1.2.1.drv' failed with exit code 1
cannot build derivation '/nix/store/jfvq63bkzchb43gc1bkga201c85xh6nz-swaglyrics-1.2.2.drv': 1 dependencies couldn't be built
error: build of '/nix/store/jfvq63bkzchb43gc1bkga201c85xh6nz-swaglyrics-1.2.2.drv' failed
```

File: pkgs/development/python-modules/ufoprocessor/default.nix (hunk 1 above)
Comment (SuperSandro2000) [reply pkgs/development/python-modules/ufoprocessor/default.nix:29]: Can people just use pytest? 

Also this is you package. Maybe you should fix that in a PR not me?

Comment (sternenseemann) [reply pkgs/development/python-modules/ufoprocessor/default.nix:29]: Sure can, wasn't really broken though, was it?

Comment (SuperSandro2000) [reply pkgs/development/python-modules/ufoprocessor/default.nix:29]: It printed to the log that zero tests where run and I am just cleaning that up. Nothing else. If you don't feel like getting the tests working I just update the comment.

**Comment by SuperSandro2000** [comment]
@siraben being tracked in https://github.com/NixOS/nixpkgs/issues/105156

File: pkgs/development/python-modules/ufoprocessor/default.nix (hunk 1 above)
Comment (sternenseemann) [reply pkgs/development/python-modules/ufoprocessor/default.nix:29]: Just give me a minute I'll have the PR done in a few minutes

Comment (sternenseemann) [reply pkgs/development/python-modules/ufoprocessor/default.nix]: #109550

**Review by jonringer — APPROVED** [review]
LGTM

Result of nixpkgs-review pr 109539 run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)<details>  <summary>22 packages built:</summary>  <ul>    <li>devpi-client</li>    <li>noto-fonts-emoji</li>    <li>python37Packages.afdko</li>    <li>python37Packages.build</li>    <li>python37Packages.check-manifest</li>    <li>python37Packages.nototools</li>    <li>python37Packages.sphinxcontrib-bibtex</li>    <li>python37Packages.ufoprocessor</li>    <li>python38Packages.afdko</li>    <li>python38Packages.build</li>    <li>python38Packages.check-manifest</li>    <li>python38Packages.nototools</li>    <li>python38Packages.sphinxcontrib-bibtex</li>    <li>python38Packages.ufoprocessor</li>    <li>python39Packages.afdko</li>    <li>python39Packages.build</li>    <li>python39Packages.nototools</li>    <li>python39Packages.sphinxcontrib-bibtex</li>    <li>python39Packages.ufoprocessor</li>    <li>swaglyrics</li>    <li>twitter-color-emoji</li>    <li>winpdb</li>  </ul></details>

**Pull request merged**